app.js

```diff
@@ -82,6 +82,7 @@ exports.init = function (isA11yTest = false, a11yTestSession = {}, ftValue) {
                 '\'sha256-G29/qSW/JHHANtFhlrZVDZW1HOkCDRc78ggbqwwIJ2g=\'',
                 'https://www.google-analytics.com',
                 'https://*.googletagmanager.com',
+                'http://*dynatrace.com',
```

Comment:
Could you, please, double check that it is supposed to be http, and not https?

Reply:
Changed from http to https , as mentioned on pcq-Dynatrace channel